.ci/Manifest.toml

```diff
@@ -1,7 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.3"
+julia_version = "1.9.2"
```

Comment:
I don't think we should change the version of Julia here, there are some tests that are failing for Julia v1.8+ in BBB